app/models/ability.rb

```diff
@@ -0,0 +1,14 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize user
```

Comment:
ở ngoài view thì có thể dùng được những method nào cancancan hỗ trợ

Reply:
Ngoài view có thể sử dụng các method:

- `can?` để kiểm tra quyền của user với action được truyền vào. Trả về true nếu user có quyền đó, còn ngược lại trả về false
VD: `can? :destroy, @project`

- `cannot?` ngược lại với can?.
VD: cannot? :destroy, @project

Reply:
sao a không thấy e dùng chỗ nào nhỉ

Reply:
Tại vì trang member cùng với trang manager đang tách biệt nhau hoàn toàn. Chỉ có phần sidebar là đang dùng chung view. Mà phần sidebar thì check bằng role của use thì hợp lý hơn nên em đang không dùng vào đâu cả. Cái cancancan em đang chỉ dùng để bắt các trường hợp mà user nhập trực tiếp url vào thôi ạ.